community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -1834,7 +1834,8 @@
         "%%writefile ./custom_container/Dockerfile\n",
         "\n",
         "# Use pytorch GPU base image\n",
-        "FROM gcr.io/cloud-aiplatform/training/pytorch-gpu.1-7\n",
+        "# FROM gcr.io/cloud-aiplatform/training/pytorch-gpu.1-7\n",
```

Comment:
Should this commented line be removed?

Reply:
Removed now. Using latest image from pre-built pytprch containers